server/views/pages/prisons/sessionTemplates/viewSingleSessionTemplate.njk

```diff
@@ -127,6 +127,15 @@
             },
             classes: "test-template-endTime"
           },
+          {
+            key: {
+              text: "Visit order restriction"
```

Comment:
```suggestion
              text: "Visit order type"
```

Personal preference; we haven't got a design for this - I just wonder if this reads better?